### PR TITLE
Final changes

### PR DESCRIPTION
**Pull request by EhrlichNati**

### CODE REVIEW

File: .idea/Garmin-Sports-Watch-Data-Cleanup.iml
Comment (erlichsefi) [inline .idea/Garmin-Sports-Watch-Data-Cleanup.iml:1]: can you add this `.idea` folder to `.gitignore`? I don't see why those configuration files need to be committed.

Comment (erlichsefi) [reply .idea/Garmin-Sports-Watch-Data-Cleanup.iml:1]: i see you already pushed the `.idea ` folder, in order that git will detect it need to remove the folder from the remote, you need.
1. copy `.idea ` in your local to different location.
2. delete the `.idea ` in local project root.
3. now git will notice the deletion and you can `add`, `commit` and `push` them.
4. add `.idea ` to `.gitignore`
5. copy the folder back to the project root.

**Review by erlichsefi — CHANGES_REQUESTED** [review]

**Comment by erlichsefi** [comment]
You didn't address my comments.

**Comment by EhrlichNati** [comment]
Fixed it now.
had some problem with the small details.

**Pull request merged**